src/components/shared/CallNodeContextMenu.js

```diff
@@ -377,16 +375,6 @@ class CallNodeContextMenu extends PureComponent<Props> {
     return funcHasRecursiveCall(thread, implementation, funcIndex);
   }
 
-  componentWillReceiveProps(nextProps: Props) {
-    if (nextProps.callNodeIndex === null && this.props.isContextMenuVisible) {
-      // If the menu was visible while callNodeIndex was
-      // changed to null, the onHide callback will not execute when
-      // null is returned below. Call _menuHidden() here to be ensure
-      // the visibility state is updated.
-      this._onHide();
-    }
```

Comment:
I was unable to reproduce the problem in the outdated comment. `_onHide` _is_ called, from somewhere else, even when the if condition evaluates to true here. So after manually testing around with this whole method removed, I concluded that it is not needed anymore.

Reply:
I think this was necessary when `callNodeIndex` was coming from the "selected" node: it was possible that the selected node was set to `null` in some situations. In that case it renders to `null` and `onHide` isn't called at unmount time. (probably they should call it at unmount if it's shown...)
But it's not happening nowadays, hopefully.